.gitignore

```diff
@@ -22,3 +22,4 @@ doc/api/
 
 # Google Secrets
 GoogleService-Info.plist
+.DS_Store
```

Comment:
I see that the **.DS_Store** file itself is still checked in. Please delete this.

Reply:
This is still not removed. Please remove this file!